### DISCUSSION TITLE
Tortuosity values from tortuosity_gdd()

### DISCUSSION THREAD

**Discussion by BrinthanK** [PMEAL/porespy#987] [Q&A]
Hi, 

We are trying to get the tortuosity valuen for the 3D tomograph using porespy.beta tortuosity_gdd function. However I am getting values that are not physical for tortuosity i.e., >> 10 (picture attached). Can you please help us in understanding why we are getting such high values for the tortuosity? 

Thank you. 

![tortuosity_values](https://github.com/user-attachments/assets/35052627-f582-409f-a4a4-4c17957ef4ed)

**Comment by jgostick** [comment]
Thst function is still a eork in progress, sorry. It's under the beta folder to be hidden for now.
